libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -18,6 +18,13 @@
 
 #define AP_BATT_MONITOR_TIMEOUT             5000
 
+#define AP_BATT_MONITOR_RES_EST_TC_1        0.5f
+#define AP_BATT_MONITOR_RES_EST_TC_2        0.1f
+
+// low voltage sources
+#define AP_BATT_MONITOR_LOW_VOLTAGE_SOURCE_RAW              0
```

Comment:
sure, changed to enum in follow-up commit.
By the way, I always worry what happens if we cast a parameter value to an enum and it's outside the valid range.  I guess I should test what happens.